app/models/voter.rb

```diff
@@ -16,11 +16,19 @@ class Voter < ApplicationRecord
     do_not_call: "Don't call back",
   }.freeze
 
+  # What's going on with this mapping? where is it getting the vote status from? GA SOS?
```

Comment:
This is for a column we don't have anymore, can remove this variable (and comment)